webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -705,13 +706,19 @@ var userOverride = {
             onConnectFailed: function (resp) {
                 console.log('onConnectFailed' + resp);
                 console.log('onconnect failed');
+                onConnectEventFailureCount += 1
```

Comment:
how do we know this failure is because of authentication failure?
what if our socket server is down or lets say we are restarting the servers. what will happen in that case, if that fails during that time?

Reply:
will need to make some changes in applozic-chat file